thriftpy2/parser/parser.py

```diff
@@ -403,7 +403,7 @@ def p_ref_type(p):
                 ref_type = resolved_ref_type
                 break
     else:
-        for index, name in enumerate(p[1].split('.')):
+        for index, name in enumerate(p[1].rsplit('.', 1)):
```

Comment:
The split count is set to one, so I thought this will not work with thrift files with multiple dot in name.

Reply:
And why change the split to rsplit? I think the resolve order is chaned and I thought it's a ad-hoc to your case and will not work with other kind of cases.